images/dnsmasq/Makefile

```diff
@@ -13,19 +13,19 @@
 # limitations under the License.
 
 VERSION ?= $(shell git describe --tags --always --dirty)
-REGISTRY ?= gcr.io/google_containers
+REGISTRY ?= gcr.io/google-containers
```

Comment:
So we are consolidating images to `google-containers` in future? Just notice that cluster-proportional-autoscaler is still using `google_containers`.